phys/module_diagnostics_driver.F

```diff
@@ -292,10 +292,18 @@ SUBROUTINE diagnostics_driver ( grid, config_flags,                  &
                ,e=grid%e                                            &
                ,sina=grid%sina                                      &
                ,cosa=grid%cosa                                      &
+               ,rho=grid%rho                                        &
```

Comment:
@smileMchen 
Ming,
Would you locate these variables under the section for input. Also a few lines down, qc,  etc.